ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -286,7 +289,10 @@ PrjFS_Result PrjFS_WritePlaceholderDirectory(
     _In_    const char*                             relativePath)
 {
 #ifdef DEBUG
-    cout << "PrjFS_WritePlaceholderDirectory(" << relativePath << ")" << endl;
+    cout
+        << "PrjFS_WritePlaceholderDirectory("
+        << relativePath
```

Comment:
Minor: You could make the printing of `relativePath` more consistent.  Here (and in `PrjFS_ConvertDirectoryToVirtualizationRoot`) it's printed inside parenthesis that follow the function name.

However, I noticed in spots below it's printed after a `:` that follows the function name.

Reply:
All of the Handle functions print the arguments with the `:` pattern while the methods that do the work use the `()` pattern. Given the nature of the arguments for the non Handle method, I think that pattern makes more sense there and the `:` pattern makes sense for Handle.

If you feel strongly, I can normalize these all though.

Reply:
Ok thanks I hand't realized that pattern.  I think it's okay to leave it that way.